spec/index.bs

```diff
@@ -849,8 +889,171 @@ A [=device=] has the following internal slots:
 
 ### Limits ### {#limits}
 
+Each <dfn dfn>limit</dfn> is a numeric limit on the usage of WebGPU on a device.
+Each [=adapter=] has a set of [=adapter/support|supported=] limits, and
+[=devices=] are created with specific limits in place.
+The device limits are enforced regardless of the adapter's limits.
+
+One limit value may be <dfn dfn for=limit>better</dfn> than another.
+A [=limit/better=] limit value always relaxes validation, enabling strictly
+more programs to be valid. For each limit, "better" is defined.
+
+Note:
+Setting "better" limits may not necessarily be desirable, as they may have a performance impact.
+Because of this, and to improve portability across devices and implementations,
+applications should generally request the "worst" limits that work for their content
+(ideally, the default values).
+
+Each limit also has a <dfn dfn for=limit>baseline</dfn> value.
+Every [=adapter=] is guaranteed to support the baseline value or [=limit/better=].
+The baseline value is also the default value for the limit in {{GPULimits}} if
+a better limit is not specified.
+
+<table class="data no-colspan-center" dfn-type=enum-value dfn-for=GPULimitName>
```

Comment:
nit: Maybe move the refactor of the table and reordering of the sections to a different PR so rebasing is simpler?